src/admin-views/admin-update-message.php

```diff
@@ -6,7 +6,8 @@
 
 ?>
 
-<p class="tribe-welcome-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
+<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 9a48dbe92857984b8bfac16f21468c92</sub>